.github/workflows/integration-test-call.yml

```
@@ -52,7 +52,15 @@ jobs:
       - name: Add success label to pull request
         if: ${{ needs.query.result == 'success' && github.event.inputs.pr_number != '' }}
         run: |
-          gh pr edit "$PR_NUM" --add-label "integration tests succeeded"
+          # Get the branch without refs/heads/
+          REF=$(git rev-parse --abbrev-ref HEAD)
+          # Only add the label if the workflow references the correct branch
+          NUM_USAGES=$(grep 'uses: github/codeql-variant-analysis-action/' .github/workflows/codeql-query.yml | grep "@${REF}$" | wc -l)
```

minor: I think this is fine, but I prefer quoting all variable assignments in bash:

```suggestion
          REF="$(git rev-parse --abbrev-ref HEAD)"
          # Only add the label if the workflow references the correct branch
          NUM_USAGES="$(grep 'uses: github/codeql-variant-analysis-action/' .github/workflows/codeql-query.yml | grep "@${REF}$" | wc -l)"
```

I don't think it matters in this case, but I do it out of habit since I've been bitten by this too many times. This would use nested quotes but that is fine.